internal/database/atendee_table.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+    "database/sql"
+    "fmt"
+    "log"
+
+    _ "github.com/go-sql-driver/mysql"
+)
+
+func CreateAttendeesTable() {
+    // Connect to the database
+    dsn := "root:admin123@tcp(localhost:3306)/events" //dsn
+    db, err := sql.Open("mysql", dsn) //open connection to db
+    if err != nil { //check for errors
+        log.Fatal(err) //exit if error is found
+    }
+    defer db.Close() //close the connection when the function ends
+	
+    // SQL query to create the attendees table
+    createTableQuery := `
+    CREATE TABLE IF NOT EXISTS attendees (
+        id INT AUTO_INCREMENT PRIMARY KEY,
+        event_id INT,
+        first_name VARCHAR(255) NOT NULL,
+        last_name VARCHAR(255) NOT NULL,
+        email VARCHAR(255) NOT NULL,
+        phone_number VARCHAR(20),
+        image_url VARCHAR(255),
```

Comment:
I am a fan of short URLs (I work at Bitly, after all 😆 ) but in theory, I think a URL can be up to 2048 characters so just to be safe, but be better to set this to VARCHAR(2048)?